src/internal/operators/isEmpty.ts

```diff
@@ -3,6 +3,60 @@ import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
 import { OperatorFunction } from '../types';
 
+/**
+ * Emits true if the input observable emits any values, or emits false if the
```

Comment:
I think this textual description says the exact opposite of what this operator does:

> Emits true if the input observable emits any values...

... and then ...

> As soon as the source Observable emits a value, `isEmpty` will emit a `true`...

But `isEmpty` emits `false` if the source emits any value. https://github.com/ReactiveX/rxjs/blob/master/src/internal/operators/isEmpty.ts#L33

Reply:
Oh oops, I got them on backwards.  Will fix.